machines/VerifiableCredential/VCItemMachine/VCItemServices.ts

```diff
@@ -197,8 +198,10 @@ export const VCItemServices = model => {
     verifyCredential: async context => {
       if (context.verifiableCredential) {
         const verificationResult = await verifyCredential(
-          context.verifiableCredential,
-          context.vcMetadata,
+          getVerifiableCredential(
```

Comment:
Since this machine is only for otp flow , isFromOpenId4VCI is always false , right?. Why don't we pass **context.verifiableCredential** directly to verifyCredential as we are doing in Issuers machine as **context.verifiableCredential?.credential** ?